h5io/_h5io.py

```diff
@@ -317,6 +323,7 @@ def _triage_write(
             where + ".csr_matrix_indices",
             cleanup_data=cleanup_data,
             slash=slash,
+            use_state=use_state,
```

Comment:
Is there a case where the `indices` are required to be an object which needs `__getstate__()` to be stored in the HDF5 file?